README.md

```diff
@@ -2450,7 +2450,7 @@ We use [Celery](http://www.celeryproject.org/) for the following general cases:
 
 We try to treat Celery as if it's just another interface to our core logic - meaning - **don't put business logic there.**
 
-Lets look at an example of a **service** that sends emails (example taken from [`Django-Styleguide-Example`](https://github.com/HackSoftware/Django-Styleguide))
+Lets look at an example of a **service** that sends emails (example taken from [`Django-Styleguide-Example`](https://github.com/HackSoftware/Django-Styleguide-Example))
```

Comment:
I think here you could put the directly file location to [`styleguide_example/emails/tasks.py`](https://github.com/HackSoftware/Django-Styleguide-Example/blob/master/styleguide_example/emails/tasks.py), where the definition and usage of sending async e-mails is implement.

What you think?

Reply:
I agree with that :+1: 

I'll change it once I merge the PR